src/ClamavValidator/ClamavValidatorServiceProvider.php

```diff
@@ -31,12 +31,13 @@ public function boot()
         $this->loadTranslationsFrom(__DIR__ . '/../lang', 'clamav-validator');
 
         $this->app['validator']
-            ->resolver(function ($translator, $data, $rules, $messages) {
+            ->resolver(function ($translator, $data, $rules, $messages, $attributes) {
```

Comment:
May be change this also to `$customAttributes` just for consistency?

Reply:
Yea, good one. I've renamed the variable and defaulted the `$customAttributes` to `[]` to prevent existing installations without the `$customAttributes` parameter from erroring on a potential `Uncaught ArgumentCountError: Too few arguments`